### PR TITLE
deprecate

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

File: docker/dempcap/build.conf [hunk 1]
```diff
@@ -1,2 +1,3 @@
 version=1.0.0
 ironbank=true
+deprecated=true
```
Comment (JudahSchwartz) [inline docker/dempcap/build.conf:3]:
```suggestion
deprecated=true
deprecated_reason=Image no longer in use by content.
```

File: docker/py2-exchangelib/build.conf [hunk 2]
```diff
@@ -1 +1,2 @@
 version=1.0.0
+deprecated=true
```
Comment (JudahSchwartz) [inline docker/py2-exchangelib/build.conf:2]:
```suggestion
deprecated=true
deprecated_reason=Image no longer in use by content
```

File: docker/pytan/build.conf [hunk 3]
```diff
@@ -1,2 +1,3 @@
 # Name value properties to use while doing a build
-version=1.0.0
\ No newline at end of file
+version=1.0.0
+deprecated=true
```
Comment (JudahSchwartz) [inline docker/pytan/build.conf:3]:
```suggestion
deprecated=true
deprecated_reason=Image no longer in use by content
```

File: docker/smb/build.conf [hunk 4]
```diff
@@ -1 +1,2 @@
 version=1.0.0
+deprecated=true
```
Comment (JudahSchwartz) [inline docker/smb/build.conf:2]:
```suggestion
deprecated=true
deprecated_reason=Image no longer in use by content
```

File: docker/datadog-api-client/build.conf [hunk 5]
```diff
@@ -1 +1,3 @@
 version=1.0.0
+deprecated=true
+deprecated_reason=Image not in use by non-deprecated content item.
```
Comment (JudahSchwartz) [inline docker/datadog-api-client/build.conf:3]:
```suggestion
```

File: docker/datadog-api-client/build.conf [hunk 6]
```diff
@@ -1 +1 @@
-version=1.0.0
+version=1.0.0
```
Comment (JudahSchwartz) [inline docker/datadog-api-client/build.conf:1]:
```suggestion
version=1.0.0

```

**Review by tkatzir — APPROVED** [review]
Good! Very important PR!

File: docker/panw-iot/build.conf [hunk 7]
```diff
@@ -1 +1,3 @@
 version=1.0.0
+deprecated=true
+deprecated_reason=Image not in use by non-deprecated content item.
```
Comment (JudahSchwartz) [inline docker/panw-iot/build.conf:3]:
```suggestion
```

**Pull request merged**